source/fundamentals/logging.txt

```diff
@@ -10,216 +10,21 @@ Logging
-
-   * - ``Server``
-     - Information related to a server instance (standalone instance,
-       ``mongos`` instance, or replica set member).
-
-   * - ``ReplSet``
-     - Information related to a replica set.
-
-   * - ``Mongos``
-     - Information related to a ``mongos`` instance.
-
-   * - ``Cursor``
-     - Information related to a cursor.
-
-   * - ``Pool``
-     - Information related to a connection pool.
-
-   * - ``Connection``
-     - Information related to a single-instance connection.
-
-   * - ``Ping``
-     - Information related to a replica set ping operation.
-
-You can add your own classes to the logger by creating your own logger
-instances as shown in the following sample code.
-
-.. code-block:: js
-
-   const { Logger } = require("mongodb");
-
-   class A {
-     constructor() {
-       this.logger = new Logger("A");
-     }
-
-     doSomething() {
-       if (this.logger.isInfo()) {
-         this.logger.info("logging A", {});
-       }
-     }
-   }
-
-   // Execute A to produce logging messages
-   const a = new A();
-   a.doSomething();
-
-The logging message is triggered when ``doSomething()`` is called and is
-presented in the following format:
-
-.. code-block:: none
-
-   [INFO-A:65156] 1585699200000 logging A {
-     type: 'info',
-     message: 'logging A',
-     className: 'A',
-     ...
-   }
-
-
-Custom Logger
--------------
-
-You can configure the behavior and format of the messages reported by the
-logger by passing in a **logger callback function** which allows you
-to access and modify the message and metadata of the logger message.
-
-The following example demonstrates how to define a logger callback
-function and add custom logic to it.
-
-.. code-block:: js
-
-   // Set debug level
-   Logger.setLevel("debug");
-
-   // Set our own logger
-   Logger.setCurrentLogger((msg, context) => {
-     // Add your custom logic here
-     context['foo'] = 'bar';
-     msg = "Hello, World! " + msg;
-
-     console.log(msg, context);
-   });
-
-   const db = client.db("sample_mflix");
-
-   // Run a command to produce logger messages
-   await db.command({ isMaster: true });
-
-The code example above adds additional information in the callback function.
-The output of the example resembles the following:
-
-.. code-block:: none
-
-   Hello, World! [DEBUG-Db:65873] 1585699200000 executing command {"isMaster":true} against sample_mflix.$cmd with options [{"readPreference":{"mode":"primary"}}] {
-     type: 'debug',
-     message: 'executing command {"isMaster":true} against sample_mflix.$cmd with options [{"readPreference":{"mode":"primary"}}]',
-     className: 'Db',
-     foo: 'bar',
-     ...
-   }
-
-For more information on the methods available on the ``Logger``, see the
-:node-api-4.0:`Logger API documentation </classes/logger.html>`.
+  const client = new MongoClient(CONNECTION_STRING, { monitorCommands:true });
+  client.on('commandStarted', (ev) => console.debug(ev));
```

Comment:
```suggestion
  client.on('commandStarted', (event) => console.debug(event));
```

Suggestion: Could event be spelled out in these? On first reading, I wasn't sure what "ev" meant (at first I thought it meant environment).